src/main/java/com/qubole/utility/RedisCache.java

```diff
@@ -184,6 +184,11 @@ public V getFromSource(K key, Callable<? extends V> valueLoader, Boolean cacheEx
     try (Jedis jedis = jedisPool.getResource()) {
       List<byte[]> valueBytes = jedis.mget(Iterables.toArray(keyBytes, byte[].class));
 
+      // check for null entry
+      if(valueBytes.size() == 1 && valueBytes.get(0) == null) {
```

Comment:
Can it be null in Line 197 as well within the for loop.

Reply:
No, that's the value returned from sourceclient i.e., metastore in our case.